pull_request_template.md

```diff
@@ -1,65 +1,28 @@
 # Summary | Résumé
 
-> 1-3 sentence description of the changed you're proposing, including a link to
-> a GitHub Issue # or Trello card if applicable.
+_TODO: 1-3 sentence description of the changed you're proposing._
 
----
+## Related Issues | Cartes liées
 
-> Description en 1 à 3 phrases de la modification proposée, avec un lien vers le
-> problème (« issue ») GitHub ou la fiche Trello, le cas échéant.
+* https://app.zenhub.com/workspaces/notify-planning-614b3ad91bc2030015ed22f5/issues/gh/cds-snc/notification-planning/1
+* https://app.zenhub.com/workspaces/notify-planning-core-6411dfb7c95fb80014e0cab0/issues/gh/cds-snc/notification-planning-core/1
 
 # Test instructions | Instructions pour tester la modification
 
-> Sequential steps (1., 2., 3., ...) that describe how to test this change. This
-> will help a developer test things out without too much detective work. Also,
-> include any environmental setup steps that aren't in the normal README steps
-> and/or any time-based elements that this requires.
-
----
-
-> Étapes consécutives (1., 2., 3., …) qui décrivent la façon de tester la
-> modification. Elles aideront les développeurs à faire des tests sans avoir à
-> jouer au détective. Veuillez aussi inclure toutes les étapes de configuration
-> de l’environnement qui ne font pas partie des étapes normales dans le fichier
-> README et tout élément temporel requis.
+_TODO: Fill in test instructions for the reviewer._
 
 # Release Instructions | Instructions pour le déploiement
 
 None.

```

Comment:
can we remove this as well?

Reply:
It is there as a default. If we remove it and there are no special instructions, the PR author should add it every time.